modules/ROOT/pages/functions/dev_guide/python/return-values-python.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies
+//
+// functions/user_guide/develop-python.adoc
+
+// [id="return-values-python_{context}"]
+= Return values
+
+Functions may return any value supported by Flask because the invocation framework proxies these values directly to the Flask server.
+
+.Example
+[source,python]
+----
+def main(context: Context):
+    data = { "message": "Howdy!" }
+    headers = { "content-type": "application/json" }
+    return body, 200, headers
+----
+
+Functions can set both headers and response codes as secondary and tertiary response values from function invocation.
```

Comment:
```suggestion
Functions can set both response codes and headers as secondary and tertiary response values from function invocation.
```
Since the response code is the second item in the return list, and headers must be the third...